optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/src/main/java/org/optaplanner/workbench/screens/domaineditor/client/widgets/planner/PlannerDataObjectEditorViewImpl.java

```diff
@@ -20,52 +20,47 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
-import com.google.gwt.core.client.GWT;
 import com.google.gwt.event.dom.client.ChangeEvent;
 import com.google.gwt.event.dom.client.ClickEvent;
-import com.google.gwt.uibinder.client.UiBinder;
-import com.google.gwt.uibinder.client.UiField;
-import com.google.gwt.uibinder.client.UiHandler;
 import com.google.gwt.user.client.ui.Composite;
-import com.google.gwt.user.client.ui.Widget;
-import org.gwtbootstrap3.client.ui.FormGroup;
-import org.gwtbootstrap3.client.ui.InlineRadio;
 import org.gwtbootstrap3.extras.select.client.ui.Select;
+import org.jboss.errai.common.client.dom.Div;
+import org.jboss.errai.common.client.dom.Input;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.EventHandler;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
 import org.kie.workbench.common.screens.datamodeller.client.util.UIUtil;
 import org.uberfire.commons.data.Pair;
 
 @Dependent
+@Templated
 public class PlannerDataObjectEditorViewImpl
         extends Composite
         implements PlannerDataObjectEditorView {
 
-    interface Binder
-            extends UiBinder<Widget, PlannerDataObjectEditorViewImpl> {
-
-    }
-
-    private static Binder uiBinder = GWT.create( Binder.class );
-
-    @UiField
-    InlineRadio notInPlanningRadioButton;
+    @Inject
+    @DataField("notInPlanningRadioButton")
```

Comment:
Not sure if you're aware, but since the field already has the name `notInPlanningRadioButton` the argument to `@DataField` isn't necessary. (Same applies to the other DataFields in this type).